ros_package_template/package.xml

```diff
@@ -32,6 +32,8 @@
 
   <buildtool_depend>catkin</buildtool_depend>
   <buildtool_depend>catkin_simple</buildtool_depend>
+  
```

Comment:
maintain this spacing in other files too.

Reply:
Shouldn't we remove other demo comments in package.xml file afterall they are highly version specific as well as catkin simple eliminates some?
If required can provide our customized guidelines